assets/cms/components/IconButton.vue

```diff
@@ -0,0 +1,70 @@
+<template functional>
+    <span class="button-container">
+        <button
+            @click='listeners.click'
+            v-bind:class="['btn', props.bootstrapClass]"
+            :disabled="props.disabled"
+            >
+            <span class="label" v-if="props.labelPosition === 'left'"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+            <Icon :icon="props.icon" :type="props.type" :color="props.disabled ? '#ccc' : props.color"/>
+
+            <span class="label" v-if="props.labelPosition === 'right' || !props.labelPosition"
+                v-bind:class="{disabled : props.disabled}">
+                <slot />
+            </span>
+
+        </button>
+    </span>
+</template>
+
+<script>
+import Icon from './Icon'
+
+export default {
+    props: {
+        outline: Boolean,
+        disabled: Boolean,
+        labelPosition: String,
+        icon: String,
+        type: String,
+        color: String,
+        bootstrapClass: {
+            type: String
+        }
+    },
+    components: {
+        Icon
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../../assets/cms/scss/bootstrap-overrides';
+.button-container {
+    button {
+        display: inline-flex;
+        align-items: center;
+        &:focus {
+            outline: none;
```

Comment:
What's this for?

Reply:
there is a blue outline from bootstrap on buttons when the button is focused, this removes it.

Reply:
Let's just use the global bootstrap styles, if you need to modify how bootstrap renders buttons do it with bootstrap-overrides or other global classes. In this case I don't think you need to remove the outline